### PR TITLE
Added some clarity around Community & Citizenship

### LINKED ISSUE #7: Clarify "Community & Citizenship" for interns
There's only one bullet there right now:

> * You do things to benefit your peers and community such as participating in meetings and providing constructive feedback on ideas and initiatives.

I agree with the feedback we received over this summer that that's not super clear. How about we change it into something like the following?:

> * You do things to benefit your peers and community such as participating in meetings, including providing constructive feedback on ideas and initiatives
> * You advocate for yourself and others when something is unclear or you believe there is an alternate approach worth discussion

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Fixes #7.

These additions are based off of talks we had offline. I don't think this is the final result of what we'd want for the Community & Citizenship competency, but it at least fixes the issue of lack of clarity around just interns IMO.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +40,8 @@ Jump to competencies for a particular level:
 
 ### Community & Citizenship
 
-- You do things to benefit your peers and community such as participating in meetings and providing constructive feedback on ideas and initiatives.
+- You do things to benefit your peers and community such as participating in meetings, including providing constructive feedback on ideas and initiatives.
+- You advocate for yourself and others when something is unclear or you believe there is an alternate approach worth discussion.
```
Comment (kipprice) [inline README.md]: I would reframe the "alternate approach" part of this to be its own line, and phrase as "You ask questions about approaches to solving problems and suggest alternate options"; "worth discussion" is IMO too subjective a statement to be useful

File: README.md [hunk 2]
```diff
@@ -66,7 +67,9 @@ Jump to competencies for a particular level:
 
 ### Community & Citizenship
 
-- You do things that benefit your peers and community such as providing feedback on product features, filing detailed bugs, and providing constructive comments.
+- You do things to benefit your peers and community such as participating in meetings, including providing constructive feedback on ideas and initiatives.
+- You advocate for yourself and others when something is unclear or you believe there is an alternate approach worth discussion.
+- You provide feedback on product features, file detailed bugs, and voice constructive comments in small- and large-group discussions.
```
Comment (kipprice) [inline README.md]: re: "constructive comments in small- and large-..."; I don't agree that this needs to be in a verbal public forum at Eng I. The way we phrased this at Epic was "You escalate early and often", with the details of how you did left up to you and your manager.

Comment (JoshuaKGoldberg) [reply README.md]: Hmm. Large-group discussions I'm on the fence about, but I would have thought small- person discussions would have thought would be an expectation. Interesting. Will ruminate...

Comment (JoshuaKGoldberg) [reply README.md]: @kipprice how about _" You escalate early and often with feedback on product features, product bug reports, and constructive criticism."_? If not, would you mind writing what you're envisioning? 😄

**Pull request merged**